src/app/explore/explore.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen, waitFor, act, fireEvent } from '@testing-library/react';
+import Explore from './page';
+import { useRouter, useSearchParams } from 'next/navigation';
+
+// Mock the fetchData function to simulate a successful API response
+jest.mock('../../../utils/fetchData', () => {
+  return {
+    __esModule: true,
+    default: jest.fn().mockResolvedValue({ hits: [] }),
+  };
+});
+
+jest.mock('next/navigation', () => ({
+  useRouter: jest.fn(),
+  useSearchParams: jest.fn(),
+}));
+
+describe('Explore Page Integration Test', () => {
+  it('should send search value to API and get a response', async () => {
```

Comment:
This seems to be testing the explore page in insolation from the API call it makes rather then testing the integration between the two.

There is also no test for an error condition.